src/utils/data/index.js

```diff
@@ -21,7 +21,9 @@ const buildFileURL = (file, recordId = null) => {
 
   const mediaPath = getMediaPath();
 
-  const rootUrl = MEDIA_ROOT_URL ? MEDIA_ROOT_URL : '/presentation';
+  const rootUrl = MEDIA_ROOT_URL 
+    ? MEDIA_ROOT_URL 
+    : window.location.origin + '/presentation';
```

Comment:
Why do we need `window.location.origin` here?

Reply:
It's needed for tldraw because it needs the full url in the "src" parameter of the asset shape that is used as the slide background. Maybe I could do that in the tldraw component instead